build.sbt

```diff
@@ -44,6 +44,15 @@ ThisBuild / libraryDependencies ++= Seq(
     TestDeps.scalatestplusScalacheck,
 )
 
+//////////////////////
+// Compiler options //
+//////////////////////
+
+ThisBuild / scalacOptions ++= Seq(
+    "-feature", // enable feature warnings
+    "-deprecation", // enable compiler warnings
+)
+
```

Comment:
We are also setting compiler flags on line 67. I think it would be best to keep them all together. What do you think?

Reply:
I had them all together, but then the options below are requirements of `scalafix` so I thought they should go under the "Linting and formatting" section. Usually I'd prefer to have them all in the same place, but this being a Scala `Seq` makes it really tempting to split them up... I guess we have to decide if we group thematically or by sbt variable.
I'm undecided, lmk what you prefer.

Reply:
If we were conditionally setting the scalac options for scalafix, I could see splitting them up, but since these options will always be enabled for all the builds, and since this section of the config is called `Compiler Options`, I'm inclined to keep them together. Already the addition of `Ypatmat-exhaust-depth` is not relevant to `scalafix` so I guess that comment and placement is just outdated?

Reply:
Good points, PTAL?